### PR TITLE
Adding test for Namespace Already Exists

### PR DESCRIPTION
**Pull request by mfl-code**
## Description

Adding test for Namespace Already Exists

## TODOs

Read the [Contribution guidelines](/.github/CODE_OF_CONDUCT.md).

- [ ] Generate the docs.
- [x] Run the relevant tests successfully.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.

## Release Notes (draft)

Adding test

### Migration Guide
N/A

### CODE REVIEW

File: internal/provider/namespace_resource_test.go [hunk 1]
```diff
@@ -47,6 +48,17 @@ resource "temporal_namespace" "test" {
 					resource.TestCheckResourceAttr("temporal_namespace.test", "owner_email", "updated@example.org"),
 				),
 			},
+			// Namespace already exists Error
```
Comment (parinmaru2) [inline internal/provider/namespace_resource_test.go:51]: Minor: we should ideally create a new test case for this scenario

**Review by parinmaru2 — APPROVED** [review]

File: internal/provider/namespace_resource_test.go (hunk 1 above)
Comment (mfl-code) [reply internal/provider/namespace_resource_test.go:68]: Moved to a new Test Case

**Review by parinmaru2 — APPROVED** [review]

**Comment by ganievs** [comment]
Hi @mfl-code thanks for contribution! Could you [sign your commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) to pass checks and LGTM!

**Comment by mfl-code** [comment]
Thank you @ganievs for approving. Can you please also merge the changes - I am not able to.

**Comment by mfl-code** [comment]
Replaced with https://github.com/platacard/terraform-provider-temporal/pull/60 for the correct commit signatures.

**Pull request closed without merging**